services/dynamic-sidecar/tests/unit/test_core_application.py

```diff
@@ -0,0 +1,28 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+import inspect
+
+from pytest_simcore.helpers.typing_env import EnvVarsDict
+from simcore_service_dynamic_sidecar.core.application import AppState, create_app
+from simcore_service_dynamic_sidecar.core.settings import DynamicSidecarSettings
+
+
+def test_create_app(mock_environment_with_envdevel: EnvVarsDict):
+    app = create_app()
+    assert isinstance(app.state.settings, DynamicSidecarSettings)
+
+
+def test_AppState_decorator_class(mock_environment_with_envdevel: EnvVarsDict):
```

Comment:
```suggestion
def test_app_state_decorator_class(mock_environment_with_envdevel: EnvVarsDict):
```

Reply:
I actually wanted to write the name of the class :-)